venues/aclweb.org/ACL/migrate_submissions_ARR_ACL.py

```diff
@@ -0,0 +1,362 @@
+
+# Save all submissions in a dictionary by paper_link
+submission_dict = {submission.content['paper_link'].split('=')[1]:submission for submission in submissions}
+
+
+# Iterate through all commitment submissions  
+for note in tqdm(notes):
+    # For each, retrieve old paper from link
+    forum = note.content['paper_link'].split('=')[1]
+    if forum not in submission_dict:
+        original_arr_sub_id = client.get_note(forum).original
+        original_arr_sub = client.get_note(original_arr_sub_id)
+        if original_arr_sub:
+            # Create new note to submit to ACL 
+            acl_sub = openreview.Note(
+                invitation="aclweb.org/ACL/2022/Conference/-/Submission",
+                readers = ["aclweb.org/ACL/2022/Conference/Program_Chairs","aclweb.org/ACL/2022/Conference"], # Should SAC be in readers? 
+                writers = ["aclweb.org/ACL/2022/Conference"],
+                signatures = ["aclweb.org/ACL/2022/Conference"],
+                content = {
+                    "paper_link": note.content["paper_link"],
+                    "paper_type":note.content["paper_type"],
+                    "track":note.content["track"],
+                    "comment":note.content.get("comment"),
+                    "authorids":original_arr_sub.content["authorids"],
+                    "authors": original_arr_sub.content["authors"],
+                    "title":original_arr_sub.content["title"],
+                    "abstract":original_arr_sub.content["abstract"],
+                    "data":original_arr_sub.content.get("data"),
+                    "software":original_arr_sub.content.get("software"),
+                    "pdf":original_arr_sub.content.get("pdf")
+                }  
+            )
+            acl_sub = client.post_note(acl_sub)
+            submission_dict[acl_sub.content['paper_link'].split('=')[1]]=acl_sub
+
+# Iterate through submission dictionary, check if each has a blind submission, and create blind submission if not 
+for forum in submission_dict.keys():
+    acl_submission = submission_dict[forum]
+    if acl_submission.forum not in blind_submissions:
+        # Post blind note with original = forum of ACL submission
+        blinded_note = openreview.Note(
+            invitation = "aclweb.org/ACL/2022/Conference/-/Blind_Submission",
+            original = acl_submission.id,
+            readers = ["aclweb.org/ACL/2022/Conference/Program_Chairs","aclweb.org/ACL/2022/Conference", "aclweb.org/ACL/2022/Conference/Paper{number}/Senior_Area_Chairs".format(number = acl_submission.number)],
+            nonreaders = ["aclweb.org/ACL/2022/Conference/Paper{number}/Conflicts".format(number = acl_submission.number)],
+            writers = ["aclweb.org/ACL/2022/Conference"],
+            signatures = ["aclweb.org/ACL/2022/Conference"],
+            content = {
+                "authorids" : "",
```

Comment:
authors and authorids are array, per convention we use the string "Anonymous" and authors group:

"authors": ["Anonymous"]
"authorids": [f"aclweb.org/ACL/2022/Conference/Paper{acl_submission.number)}/Authors"]